client/components/demo/index.ts

```diff
@@ -1,8 +1,10 @@
 import Demo from './Demo/Demo';
 import DemoPreview from './DemoPreview/DemoPreview';
+import DevToolsClone from './DevToolsClone/DevToolsClone';
```

Comment:
Let's put this in a different folder. The `demo` folder is only related to components we'll feature on the `/demo` pages. Like [https://frontend.ro/demo/html/atributul-start-pentru-liste-ordonate](https://frontend.ro/demo/html/atributul-start-pentru-liste-ordonate)

Reply:
+1